first_hour_on_vwb/widget_utils.py

```diff
@@ -101,4 +101,16 @@ def __post_init__(self):
         )
     
     def get(self):
-        return self.button
\ No newline at end of file
+        return self.button
+    
+class ShowOptionalCheckbox():
+    def __init__(self):
+        self.checkbox = widgets.Checkbox(
+            False, 
+            description = "Show optional parameters",
```

Comment:
While we're here, lets fixup the spacing around equal signs in this file to be consistent.

Reply:
I have installed and run the Google Python linter/formatter on this file now, so that it's self-consistent.